docker/e2e/cypress/integration/migrating_users.spec.js

```diff
@@ -104,22 +91,17 @@ describe('user migration plugin', () => {
      * If fields are not populated here, the will not be visible in user account (KC Bug??)
      */
     function writeAdminPersonalInfo() {
-        cy.intercept('GET', '/resources/**/admin/keycloak/partials/user-list.html')
-            .as("userList");
-        cy.visit('/admin/master/console/#/realms/master/users');
-        cy.wait('@userList');
-        getAllUsers();
-        cy.get('td').contains(ADMIN_USERNAME)
-            .should('have.length.gte', 0).then(userElement => {
-            if (!userElement.length) {
-                return;
-            }
-            cy.wrap(userElement).parent().contains('Edit').click();
-            cy.get('#email').clear().type(ADMIN_EMAIL);
-            cy.get('#firstName').clear().type(ADMIN_USERNAME);
-            cy.get('#lastName').clear().type(ADMIN_USERNAME);
-            cy.get('form[name="userForm"]').get('button[type="submit"]').contains('Save').click({force: true});
-        });
+        cy.visit('/admin/master/console/#/master/users');
+        cy.url().should('include', '/admin/master/console/#/master/users');
+        cy.get('.pf-c-input-group').get('input').clear().type('*');
+        cy.get('.pf-c-input-group').get('.pf-c-button.pf-m-control').click();
+        cy.get('table').get('td[data-label="Username"]').get('a').contains(ADMIN_USERNAME).click();
+        cy.get('#kc-email').clear();
+        cy.wait(500);
```

Comment:
This seems flakey. Are we sure there is no other way to do this than using `cy.wait(500)`?
If not, then I think a comment could be helpful explaining why the `wait` is needed.